src/pages/messages/components/NviCandidatesNavigationAccordion.tsx

```diff
@@ -94,157 +60,77 @@ export const NviCandidatesNavigationAccordion = () => {
+              <Box sx={{ display: 'flex', gap: '0.5rem' }}>
+                <LinearProgress
+                  aria-labelledby="progress-label"
+                  variant="determinate"
+                  value={nviCompletedPercentage}
+                  sx={{
+                    flexGrow: '1',
+                    my: '0.175rem',
+                    height: '1rem',
+                    bgcolor: 'white',
+                    borderRadius: '0.5rem',
+                    '& .MuiLinearProgress-bar': {
+                      backgroundColor: 'info.main',
+                    },
+                  }}
+                />
+                <Typography sx={{ textAlign: 'center' }}>{nviCompletedPercentage}%</Typography>
+              </Box>
               <Typography id="progress-label" gutterBottom>
                 {t('tasks.nvi.completed_count', {
-                  completed: nviCandidatesCompeted,
+                  completed: nviCandidatesCompleted,
                   total: nviCandidatesTotal,
                 })}
               </Typography>
-              <LinearProgress
-                aria-labelledby="progress-label"
-                variant="determinate"
-                value={nviCompletedPercentage}
-                sx={{
-                  my: '0.175rem',
-                  height: '0.75rem',
-                  bgcolor: 'white',
-                }}
-              />
-              <Typography sx={{ textAlign: 'center' }}>{nviCompletedPercentage} %</Typography>
 
-              <Box sx={{ display: 'flex', justifyContent: 'center', mt: '0.5rem' }}>
-                <MuiLink
-                  data-testid={dataTestId.tasksPage.nvi.toggleStatusLink}
-                  component={Link}
-                  to={{
-                    pathname: isOnNviCandidatesPage ? UrlPathTemplate.TasksNviStatus : UrlPathTemplate.TasksNvi,
-                    search: `?${NviCandidatesSearchParam.Year}=${nviParams.year}`,
-                  }}>
-                  {isOnNviCandidatesPage ? t('tasks.nvi.show_reporting_status') : t('tasks.nvi.show_candidate_search')}
-                </MuiLink>
-              </Box>
+              <ButtonGroup
+                sx={{
+                  display: 'flex',
+                  flexDirection: 'column',
+                  gap: '0.5rem',
+                  justifyContent: 'center',
+                  mt: '0.5rem',
+                }}>
+                <SelectableButton
+                  data-testid={dataTestId.tasksPage.nvi.showCandidateSearchButton}
+                  sx={{ justifyContent: 'center' }}
+                  isSelected={isOnNviCandidatesPage}
+                  onClick={() => navigate(UrlPathTemplate.TasksNvi)}>
+                  {t('tasks.nvi.show_candidate_search')}
+                </SelectableButton>
+                <SelectableButton
+                  data-testid={dataTestId.tasksPage.nvi.showReportingStatusButton}
+                  sx={{ justifyContent: 'center' }}
```

Comment:
Trenger vi justifyContent om knappene skal ha fullWidth som de har nå?
```suggestion
                  isSelected={isOnNviCandidatesPage}
                  onClick={() => navigate(UrlPathTemplate.TasksNvi)}>
                  {t('tasks.nvi.show_candidate_search')}
                </SelectableButton>
                <SelectableButton
                  data-testid={dataTestId.tasksPage.nvi.showReportingStatusButton}
```